src/frontend/packageJson.test.ts

```diff
@@ -5,6 +5,7 @@ import * as semver from 'semver';
 describe('frontend package.json', () => {
   let frontendMapeoDependencies: Record<string, string>;
   let backendMapeoDependencies: Record<string, string>;
+  let allDependencies: Record<string, string>;
```

Comment:
would rename this to `allFrontendDependencies` (and rename other related variables accordingly)

Reply:
https://github.com/digidem/comapeo-mobile/pull/893/commits/7993eb9ff21ad00e84957db58012f9a637f3f772